sdb/stocksdashboard.py

```diff
@@ -206,17 +207,16 @@ def plot_stock(self, input_data=None, p=None, column='adj_close',
 
         data, names = Formatter().format_data(input_data)
         colors = get_colors(len(data))
-        params = self.update_params(params, kwargs, names)
-
+        params = self.update_params(params=params, names=names)
```

Comment:
I don't agree with this change, I think we should be able to setup a set of parameters in the building of the dashboard aside from the particular aesthetics.